### PR TITLE
V0.23.2

### PR DESCRIPTION
**Pull request by schancel**

### CODE REVIEW

File: contrib/aur/bitcoin-abc-qt/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Maintainer: Josh Ellithorpe <quest@mac.com>
```
Comment (schancel) [inline contrib/aur/bitcoin-abc-qt/PKGBUILD:1]: Needs to be deleted.

File: contrib/devtools/chainparams/abc/chainparams_main.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-00000000000000005ce0908e5a27e2f4c202f9f44445afff0899d254439f8480
-00000000000000000000000000000000000000000153f1c5a543971b7f71bff5
+00000000000000001f9f06b89da888cfbb697a3c06251031e46e1c5f9cab259d
```
Comment (schancel) [inline contrib/devtools/chainparams/abc/chainparams_main.txt:1]: Need to investigate this.

File: src/networks/abc/chainparamsconstants.cpp [hunk 3]
```diff
@@ -5,8 +5,8 @@
 #include <chainparamsconstants.h>
 
 namespace ChainParamsConstants {
-    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("00000000000000005ce0908e5a27e2f4c202f9f44445afff0899d254439f8480");
-    const uint256 MAINNET_MINIMUM_CHAIN_WORK = uint256S("00000000000000000000000000000000000000000153f1c5a543971b7f71bff5");
+    const BlockHash MAINNET_DEFAULT_ASSUME_VALID = BlockHash::fromHex("00000000000000001f9f06b89da888cfbb697a3c06251031e46e1c5f9cab259d");
```
Comment (schancel) [inline src/networks/abc/chainparamsconstants.cpp:8]: Probably wrong, not sure if we had updated this before? @EyeOfPython

Comment (EyeOfPython) [reply src/networks/abc/chainparamsconstants.cpp:8]: No, we haven't.

**Review by EyeOfPython — APPROVED** [review]
Seems like the changes sum up to:
- Updated mainnet chainwork
- Moved from Travis to Cirrus CI
- Lots of secp256k1 changes from core
- Lots cashtab changes

The chainwork changes are a bit annoying, but I'd say we just keep them to make it easier to backport later on. Once we have our own mainnet chainwork we will have to be more careful.

**Pull request merged**